internal/util/importutilv2/parquet/field_reader.go

```diff
@@ -408,8 +405,8 @@ func ReadBinaryData(pcr *FieldReader, count int64) (any, error) {
 			}
 		case arrow.LIST:
 			listReader := chunk.(*array.List)
-			if !isRegularVector(listReader.Offsets(), pcr.dim, true) {
-				return nil, merr.WrapErrImportFailed("binary vector is irregular")
+			if !isVectorAligned(listReader.Offsets(), pcr.dim, dataType) {
+				return nil, merr.WrapErrImportFailed("binary vector not aligned")
```

Comment:
please refine the error to "xxx vector not aligned"

Reply:
updated